deepset_cloud_sdk/workflows/sync_client/files.py

```diff
@@ -44,11 +44,12 @@ def upload(
     timeout_s: Optional[int] = None,
     show_progress: bool = True,
     recursive: bool = False,
+    desired_file_types: str = ".txt, .pdf",
```

Comment:
Could we use a list of strings here, make sure that every element is part of the allowed list (and raise an error if not) and only do the parsing within the CLI method ? (The SDK can handle the list of stings i think :) )

Reply:
Unfortunately, when passing a list of strings on the command line, like  `--desired_file_types [.txt, .pdf]`, all elements inside got rendered as one string, e.g. `[".txt, .pdf"]`.